tap_google_ads/report_definitions.py

```diff
@@ -886,6 +886,144 @@
     "metrics.video_view_rate",
     "metrics.video_views",
     "metrics.view_through_conversions",
+    "segments.date",
+    "segments.day_of_week",
+    "segments.hour",
+    "segments.month",
+    "segments.month_of_year",
+    "segments.quarter",
+    "segments.week",
+    "segments.year",
+]
+CAMPAIGN_PERFORMANCE_BY_AD_NETWORK_TYPE_REPORT_FIELDS = [
```

Comment:
Can't it be done via `exclude_fields` in `couplerio/importers/google_ads/exclude_fields.py` ?